app/models/health/patient.rb

```diff
@@ -870,17 +870,17 @@ def qualified_activities_since date: 1.months.ago
 
     # This does not return a scope
     def valid_qualified_activities_since date: 1.months.ago
-      qualified_activities_since(date: date).to_a.select(&:compute_procedure_valid?)
```

Comment:
This was `to_a` because it was computing procedure valid.  If that doesn't need to happen, I think it would be better to return a scope.

Reply:
But, we'd need to confirm downstream that a scope would be ok.

Reply:
Confirmed the only use is to call `count` on them, so we can remove `to_a`